Sky Atlas/Sky Atlas.html

```diff
@@ -11706,17 +11881,41 @@ <h1>Sections & Primary Docs</h1>
           <td>
             <dfn>A.2.10 - Initial Cash Grant To Spark Foundation</dfn>
           </td>
-          <td>Initial Cash Grant To Spark Foundation </td>
+          <td>Initial Cash Grant To Spark Foundation</td>
           <td>Core</td>
           <td><p>The founding team of Spark has proposed a cash grant of 800,000 USDS per month to the Spark Foundation from Spark's Genesis Capital Allocation for a three (3) month period, beginning at the time of the Genesis Capital Allocation. The purpose of the grant is to enable Spark Foundation to fulfill its purpose of promoting the growth and development of Spark. This funding will support essential activities such as engineering efforts, community engagement, research, infrastructure, and administrative operations.</p><p>Sky Governance hereby consents to this cash grant. The first month's transfer must be made to the Spark Foundation immediately after the transfer of the Genesis Capital Allocation. See <dfn>A.2.10 - Transfer Of Genesis Capital Allocation To Spark SubProxy</dfn>. Transfers for subsequent months will be made proportionally in Spark Spells included in Sky Executive Votes unless otherwise agreed by Sky and Spark.</p></td>
         </tr>
+        <tr>
+          <td>
+            <dfn>A.2.10 - Subsequent Allocation Mechanism</dfn>
+          </td>
+          <td>Subsequent Allocation Mechanism</td>
+          <td>Core</td>
+          <td>After the initial cash grant (see <def>A.2.10 - Initial Allocation Mechanism</def>) but before the SPK or GROVE token is decentralized enough to allow meaningful governance by token holders, the respective founding teams may propose subsequent cash grants to their Prime Foundations. Sky must consent to each subsequent cash grant via approving the Sky Atlas modification.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>A.2.10 - Subsequent Cash Grant To Spark Foundation</dfn>
+          </td>
+          <td>Subsequent Cash Grant To Spark Foundation</td>
+          <td>Core</td>
+          <td><p>he founding team of Spark has proposed a cash grant of 1,100,000 USDS per month to the Spark Foundation from Spark’s Prime Treasury for a three (3) month period, beginning on October 1, 2025. The purpose of this grant is to enable Spark Foundation to fulfill its purpose of promoting the growth and development of Spark. This funding will support essential activities such as engineering and product development, community engagement and growth initiatives, research and governance contributions, infrastructure and operational maintenance, and administrative operations.</p><p>Sky Governance hereby consents to this cash grant. The transfer for October must be made to the Spark Foundation in a Spark Spell included in the October 2, 2025 Executive Vote. Transfers for subsequent months will be made proportionally in Spark Spells included in Sky Executive Votes unless otherwise agreed by Sky and Spark.</p></td>
+        </tr>
+         <tr>
```

Comment:
nit: indentation is off here

```suggestion
        <tr>
```